rocky/reports/views/generate_report.py

```diff
@@ -115,7 +122,7 @@ def get(self, request, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["available_report_types"] = self.get_report_types(get_report_types_for_oois(self.selected_oois))
+        context["available_report_types"] = self.available_report_types
```

Comment:
Perhaps I don't understand the code enough yet, but can't we add lines 112-114 here? We should avoid stacking `setup()` calls where possible since it obfuscates the flow of control a lot I find